frontend/src/scenes/session-recordings/saved-playlists/SavedSessionRecordingPlaylists.tsx

```diff
@@ -223,10 +232,31 @@ export function SavedSessionRecordingPlaylists({ tab }: SavedSessionRecordingPla
 
     return (
         <div className="deprecated-space-y-4">
+            <LemonBanner type="info" dismissKey="session-recordings-playlists-banner">
+                We've made some updates!
+                <br />
+                <p className="font-normal">
+                    Playlists used to combine saved filters and pinned recordings, but that sometimes led to confusion.
+                    Now, filters and recordings are handled separately:
```

Comment:
```suggestion
                    Now, they are handled separately:
```

or "pinned recordings"